src/main/java/nb/kafka/operator/KafkaUtilities.java

```diff
@@ -228,16 +231,22 @@ private void addAcl(Collection<AclBinding> acls, Collection<AclBinding> effectiv
   }
 
   TopicDescription topicDescription(String topicName) {
+    count = 0;
     DescribeTopicsResult dt = adminClient.describeTopics(singleton(topicName));
-    try {
-      dt.all().get();
-      TopicDescription topicDescription = dt.values().get(topicName).get();
-      return topicDescription;
-    } catch (InterruptedException | ExecutionException  e) {
-      if (e.getCause() != null && e.getCause() instanceof UnknownTopicOrPartitionException) {
-        return null;
+    while(true){
```

Comment:
Looks good, can you just follow coding convention: space before brackets, braces:
`while (true) {`
and also encapsulate all if - else blocks in braces like:
```
if (++count == maxTries) {
...
} else {
...
}
```